tests/matchers/test_numbers.py

```diff
@@ -26,13 +28,14 @@ def test_failure_message(self):
     def test_register(self):
         expect(expect.matcher('below')) == Below
 
+
 class TestWithin(unittest.TestCase):
     def test_matches(self):
-        expect(Within(1, 0, 2).matches()) == True
-        expect(Within(2, 3, 4).matches()) == False
+        expect(Within(1, 0, False, 2).matches()) == True
+        expect(Within(2, 3, False, 4).matches()) == False
```

Comment:
@catriuspham You are right, but I think we can't do it here. Because this `Within` was extended from `Base`, which is actually defined like this:
```python
class Base:
  def __init__(actual, expected, is_negated, *args):
    pass
```
The last value of `Within` is actually the `agrs[0]`, then we can't control it.